server/views/pages/checkTransfer.njk

```diff
@@ -1,75 +1,97 @@
-                            <dd class="govuk-summary-list__value" data-qa="prisoner-name">
-                                {{ data.firstName}} {{ data.lastName }}
-                            </dd>
-                        </div>
-                        <div class="govuk-summary-list__row">
-                            <dt class="govuk-summary-list__key">
-                                Date of birth
-                            </dt>
-                            <dd class="govuk-summary-list__value" data-qa="dob">
-                                {{ data.dateOfBirth | formatDate('D MMMM YYYY') }}
-                            </dd>
-                        </div>
-                        {% if data.prisonNumber %}
-                        <div class="govuk-summary-list__row">
-                            <dt class="govuk-summary-list__key">
-                                Prison number
-                            </dt>
-                            <dd class="govuk-summary-list__value" data-qa="prison-number">
-                                {{ data.prisonNumber }}
-                            </dd>
-                        </div>
-                        {% endif %}
-                        {%  if data.pncNumber %}
-                        <div class="govuk-summary-list__row">
-                            <dt class="govuk-summary-list__key">
-                                PNC number
-                            </dt>
-                            <dd class="govuk-summary-list__value" data-qa="pnc-number">
-                                {{ data.pncNumber }}
-                            </dd>
+                    <span class="govuk-caption-xl"> Confirm a prisoner's arrival</span>
+                    <h1 class="govuk-heading-l">This person is being transferred from another establishment</h1>
+                    <p>
+                        {{ data.firstName }} {{ data.lastName }} will be booked into this establishment using their existing prisoner record.
+                    </p>
+                    <h2 class="govuk-heading-m">Details from prisoner record</h2>
+                        <div class = "prisoner-record-detail prisoner-header govuk-!-margin-bottom-8">
+                            <div>
+                                {{ govukSummaryList({
+                                attributes: {'data-qa':"list" },
+                                classes: "govuk-!-margin-bottom-0",
+                                rows: [
+                                    {
+                                    key: {
+                                        text: "Name",
+                                        classes: "no-border-bottom govuk-!-padding-bottom-0"
+                                    },
+                                    value: {
+                                        text: data.firstName + ' ' + data.lastName,
+                                        classes: "no-border-bottom govuk-!-padding-bottom-0 data-qa-name"
```

Comment:
set 'data-qa-*' as a class solely for integration tests. Govuk summary list component doesn't have options to set data attributes for summary rows.